src/chapters/booleans-and-conditionals/exercises.rst

```diff
@@ -42,8 +42,13 @@ Prompt the user to enter a number:
    Else print, ``"Number too small."``
 #. Check if ``user_entry`` is all lowercase. If so, print, ``"___ is all
    lowercase."`` Else, print, ``"___ is NOT all lowercase."`` Fill in the blank
-   with the value of ``user_entry``. (*Hint*: Take advantage of the
-   ``.lower()`` string method).
+   with the value of ``user_entry``. 
+	
+.. admonition:: Tip 
+
+	To check if a string is all lowercase, you can use the string method ``.lower()``. We haven't discussed how to use this yet,
+	so read about it `here <https://www.w3schools.com/python/ref_string_lower.asp>`__ and use the *Try it Yourself* button to 
+	explore how to use the method. 
```

Comment:
Oops.  That's what happens when you write chapter 5 before anything else.  (I had a mental note to add a link, but that obviously didn't happen).

There are some quick ``.lower()`` examples in the string method appendix, just in case we want to use an internal link instead of an external one.

On second thought, maybe we should just re-write this exercise to avoid `.lower()` altogether.  Opinion?

Reply:
OK.